src/components/templates/Search/utils.ts

```diff
@@ -66,42 +66,72 @@ export function getSearchQuery(
       `(service.attributes.additionalInformation.categories:\"${categories}\")`
     : text || ''
 
+  if (searchTerm.startsWith('did:op:')) {
```

Comment:
why do we need this?

Reply:
Removed it because searching by id that included "did:op:" in the beginning gave no result. Since the address that follows differentiates the assets, I thought that could be a solution.

Reply:
it should work without removing did:op. Comment it for now. It work ok in my tests without removing `did:op`

Reply:
searching for `a2B8b3aC4207CFCCbDe4Ac7fa40214fd00A2BA71` is not working anyway